src/CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ function(valhalla_module)
      PRIVATE
        ${VALHALLA_BINARY_DIR}
        ${VALHALLA_BINARY_DIR}/valhalla)
+  
+  target_link_libraries(${library} Boost::headers)
```

Comment:
it seems to also work with `boost::boost` as link. is this the right way to do it? conan cmake seems to do it like this themselves for tests: https://github.com/conan-io/conan-center-index/blob/dc40d949a49ac8e04f3a4020a0b208c1795680f5/recipes/boost/all/test_package/CMakeLists.txt#L127